framework/console/controllers/BaseMigrateController.php

```diff
@@ -624,25 +727,42 @@ protected function migrateToVersion($version)
     protected function getNewMigrations()
     {
         $applied = [];
-        foreach ($this->getMigrationHistory(null) as $version => $time) {
-            $applied[substr($version, 1, 13)] = true;
+        foreach ($this->getMigrationHistory(null) as $class => $time) {
+            $applied[trim($class, '\\')] = true;
+        }
+
+        $migrationPaths = [];
+        if (!empty($this->migrationPath)) {
+            $migrationPaths[''] = $this->migrationPath;
```

Comment:
Maybe, it's better to do this operation in `init()` method and mark `migrationPath` property as deprecated to drop it in 2.1.x?